pkg/sql/plan/build_insert.go

```diff
@@ -368,33 +368,19 @@ func getInsertTable(stmt tree.TableExpr, ctx CompilerContext) (*ObjectRef, *Tabl
 	}
 }
 
-func BuildIndexInfos(ctx CompilerContext, dbName string, defs []*plan.TableDef_DefType) []*plan.IndexInfo {
+func buildIndexDefs(defs []*plan.TableDef_DefType) (*UniqueIndexDef, *SecondaryIndexDef) {
+	var uIdxDef *UniqueIndexDef = nil
+	var sIdxDef *SecondaryIndexDef = nil
 	for _, def := range defs {
-		if idxDef, ok := def.Def.(*plan.TableDef_DefType_Idx); ok {
-			infos := make([]*plan.IndexInfo, 0)
-			idx := idxDef.Idx
-
-			for i := range idx.IndexNames {
-				_, tableDef := ctx.Resolve(dbName, idx.TableNames[i])
-				info := &plan.IndexInfo{
-					TableName: idx.TableNames[i],
-					Cols:      make([]*plan.ColDef, 0),
-					ColNames:  make([]string, 0),
-					Field:     &plan.Field{ColNames: idx.Fields[i].ColNames},
-				}
-				if tableDef.CompositePkey != nil {
-					info.Cols = append(info.Cols, tableDef.CompositePkey)
-					info.ColNames = append(info.ColNames, tableDef.CompositePkey.Name)
-				}
-				for _, col := range tableDef.Cols {
-					info.Cols = append(info.Cols, col)
-					info.ColNames = append(info.ColNames, col.Name)
-				}
-				infos = append(infos, info)
-
+		if idxDef, ok := def.Def.(*plan.TableDef_DefType_UIdx); ok {
+			uIdxDef = idxDef.UIdx
+			for _, f := range uIdxDef.Fields {
+				println("bbb index def", f.Cols[0].Name)
```

Comment:
The 'println' function can only appear in the service startup phase and debugging. After the service runs, it cannot be written like this。

Reply:
my fault, I will delete this.